packages/api-rx/test/e2e/queries.spec.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017-2018 @polkadot/api-rx authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import testingPairs from '@polkadot/util-keyring/testingPairs';
+
+import Api from '../../src';
+
+const keyring = testingPairs();
+
+describe.skip('e2e queries', () => {
+  let api;
+
+  beforeEach(async () => {
+    api = await Api.create().toPromise();
```

Comment:
could rename `Api` to `ApiRx` on line 7 and 15?

Reply:
As per comments above, no need for the naming, we only have one here.